frameworks/digital-outcomes-and-specialists-5/questions/declaration/subcontracting.yml

```diff
@@ -1,5 +1,5 @@
 name: Subcontractors or consortia
-question: "Please confirm whether you will provide Digital Outcomes and Specialists&nbsp;5 services:"
+question: "Can you confirm whether you will provide Digital Outcomes and Specialists&nbsp;5 services:"
```

Comment:
There are lots of uses of 'please' in the declaration questions, should we be changing those as well?

Reply:
Yes, all they should all start with 'Can you' - if it's doable please.

Reply:
Handled in https://github.com/alphagov/digitalmarketplace-frameworks/pull/644